WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -3177,4 +3156,19 @@ extension AztecPostViewController {
         static let acceptTitle  = NSLocalizedString("Yes", comment: "Yes")
         static let cancelTitle  = NSLocalizedString("Not Now", comment: "Nicer dialog answer for \"No\".")
     }
+
+    struct UnsavedChangesAlert {
```

Comment:
Aside from the private discussion we had on this, we should at least mark this private.